usagov/vote-gov-drupal
# VOTE-1486 Add Arabic language, config and string translations

## Jira ticket (required)
https://bixal-projects.atlassian.net/browse/VOTE-1486

## Description (optional)
Add the Arabic language (disabled), configuration for meta tags and date format and string translations

## Deployment and testing (required, if applicable)
### Post-deploy steps
1. `lando retune`

### Testing steps
1. Visit http://vote-gov.lndo.site/admin/config/regional/language confirm Arabic language is initialized and disabled
2. Visit http://vote-gov.lndo.site/admin/config/search/metatag/node__state_territory/translate/ar/edit and confirm Arabic metatags have been created
3. Visit http://vote-gov.lndo.site/admin/config/regional/date-time/formats/manage/revision_date/translate/ar/edit confirm Arabic date format has been created
4. Visit http://vote-gov.lndo.site/admin/config/regional/translate, filter by Arabic translated strings only and confirm string translations are populated.
